langgenius/dify
# feat: add support for AWS Bedrock Mistral AI model

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of Change

Please delete options that are not relevant.

# How Has This Been Tested?
1. Create new chat APP
2. Select any model of Mistral AI of Bedrock
3. Chat with it, see if it can response correct.

# Suggested Checklist:
